thrift.js

```diff
@@ -111,6 +111,8 @@ Thrift.prototype._init = function _init(options) {
     this.asts = options.asts || Object.create(null);
     // filename to Thrift instance
     this.memo = options.memo || Object.create(null);
+    // file encoding
+    this.encoding = options.encoding || 'ascii'
```

Comment:
This line is duplicated on line 128 with `utf-8` instead of `ascii`.

Reply:
addressed. ty.